pkg/controller/utils/utils.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"encoding/json"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+)
+
+func YamlToJSONIAMPolicy(role awsv1alpha1.AWSFederatedRole) (string, error) {
```

Comment:
I think this function should be named structToJSONIAMPolicy or better yet  MarshalIAMPolicy. Since the func is not converting yaml but a struct to json.

Reply:
It's converting Yaml from the CR

Reply:
I like marshall though